extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3616,4 +3616,16 @@ static public function unserializeHandler( $errno, $errstr ) {
 	public static function clearVariablesCache() {
 		static::$variablesCache = [];
 	}
+
+	/**
+	 * Temporary helper when we need to handle something for a UCP wiki
+	 * in the current app.
+	 *
+	 * @param  int     $wikiId
+	 * @return boolean
+	 */
+	public static function isUCPWiki( int $wikiId ): bool {
```

Comment:
I've created similiar function: https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiFactory/WikiFactory.php#L3642-L3655, let's remove it or use it here too.